src/main/java/dev/ai4j/openai4j/DefaultOpenAiClient.java

```diff
@@ -41,6 +42,7 @@ public DefaultOpenAiClient(String apiKey) {
     private DefaultOpenAiClient(Builder serviceBuilder) {
         this.baseUrl = serviceBuilder.baseUrl;
         this.apiVersion = serviceBuilder.apiVersion;
+        this.userAgent = serviceBuilder.userAgent;
```

Comment:
new member field userAgent is not used
so it's probably not needed

Reply:
Indeed I messed up! But that's already merged, so I'll do another PR